core/server/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -658,19 +661,19 @@ public StartupConsistencyCheck getStartupConsistencyCheck() {
    * @throws AccessControlException if permission checking fails
    */
   public long getFileId(AlluxioURI path) throws AccessControlException {
-    long flushCounter = AsyncJournalWriter.INVALID_FLUSH_COUNTER;
+    Mutable<Long> flushCounter = new MutableObject<>(AsyncJournalWriter.INVALID_FLUSH_COUNTER);
```

Comment:
- yes, flush counter should be acquired before locked path. Agree.

- for me, fine with 4)